packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -440,13 +440,23 @@ export const transformers = (model: ModelData): string[] => {
 	}
 
 	if (model.pipeline_tag && LIBRARY_TASK_MAPPING.transformers?.includes(model.pipeline_tag)) {
-		const pipelineSnippet = [
-			"# Use a pipeline as a high-level helper",
-			"from transformers import pipeline",
-			"",
+		let pipelineSnippet = ["# Use a pipeline as a high-level helper", "from transformers import pipeline", ""];
+
+		if (model.tags?.includes("conversational")) {
```

Comment:
I suppose there's no way to ensure the tokenizer has a chat template, is there?

Reply:
`model.config?.tokenizer_config?.chat_template` should work 👀

Reply:
Yes, but as discussed offline before this PR, `transformers` has a default chat template if not available

Reply:
Yes, but there was a previous conversation where some problems were exposed and it resulted in https://github.com/huggingface/transformers/pull/30346. Internal thread: https://huggingface.slack.com/archives/C01N44FJDHT/p1713535581014499?thread_ts=1713516443.068489&cid=C01N44FJDHT

Reply:
(Just providing more context, I'm totally fine with merging as is and dealing with changes later)

Reply:
Good point, I wasn't aware it was deprecated. I added the extra check for the template!